src/praisonai/tests/unit/test_autogen_version_selection.py

```
@@ -0,0 +1,386 @@
+"""
+AutoGen Version Selection Tests
+
+This test module focuses specifically on testing the version selection logic
+and environment variable handling for AutoGen v0.2 and v0.4 support.
+"""
+
+import pytest
+import os
+import sys
+from unittest.mock import Mock, MagicMock, patch
```

Ruff (0.12.2)

`unittest.mock.MagicMock` imported but unused

Remove unused import: `unittest.mock.MagicMock`

(F401)